integration-tests/src/test/java/oracle/kubernetes/operator/ShutdownOptionsBase.java

```diff
@@ -0,0 +1,165 @@
+        testAppUrl.append("weblogic/");
+      }
+      testAppUrl.append(testAppPath);
+      // curl cmd to call webapp
+
+      StringBuffer webServiceUrl = new StringBuffer("curl --silent --noproxy '*' ");
+      webServiceUrl
+          .append(" -H 'host: ")
+          .append(domain.getDomainNs())
+          .append(".org' ")
+          .append(testAppUrl.toString());
+
+      // Send a HTTP request to keep open session
+      String curlCmd = webServiceUrl.toString();
+      ExecCommand.exec(curlCmd);
+    }
+  }
+
+  /**
+   * Shutdown managed server and returns spent shutdown time.
+   *
+   * @throws Exception If failed to shutdown the server.
+   */
+  protected static long shutdownServer(String serverName, String domainNS, String domainUid) throws Exception {
+    long startTime;
+    startTime = System.currentTimeMillis();
+    String cmd = "kubectl delete pod " + domainUid + "-" + serverName + " -n " + domainNS;
+    LoggerHelper.getLocal().log(Level.INFO, "command to shutdown server <" + serverName + "> is: " + cmd);
+    ExecResult result = ExecCommand.exec(cmd);
+    long terminationTime = 0;
+    if (result.exitValue() != 0) {
+      throw new Exception("FAILURE: command " + cmd + " failed, returned " + result.stderr());
+    }
+    TestUtils.checkPodCreated(domainUid + "-" + serverName, domainNS);
+    long endTime = System.currentTimeMillis();
+    terminationTime = endTime - startTime;
+    return terminationTime;
+  }
+
+  /**
+   * Check the applied properties are set in the pod.
+   *
+   * @param podName - name of the pod
+   * @param domainNS - domain namespace
+   * @param props - set of properties to check
+   * @throws Exception If failed verify the property update.
+   */
+  protected static boolean checkShutdownUpdatedProp(String podName, String domainNS, String... props)
+      throws Exception {
+
+    HashMap<String, Boolean> propFound = new HashMap<String, Boolean>();
+    StringBuffer cmd = new StringBuffer("kubectl get pod ");
+    cmd.append(podName);
+    cmd.append(" -o yaml ");
+    cmd.append(" -n ").append(domainNS);
+    cmd.append(" | grep SHUTDOWN -A 1 ");
+
+    LoggerHelper.getLocal().log(Level.INFO,
+        " Get SHUTDOWN props for " + podName + " in namespace " + " with command: '" + cmd + "'");
+
+    ExecResult result = ExecCommand.exec(cmd.toString());
+    String stdout = result.stdout();
+    LoggerHelper.getLocal().log(Level.INFO, "Output " + stdout);
+    boolean found = false;
+    for (String prop : props) {
+      if (stdout.contains(prop)) {
+        LoggerHelper.getLocal().log(Level.INFO, "Property with value " + prop + " has found");
+        propFound.put(prop, new Boolean(true));
+      }
+    }
+    if (props.length == propFound.size()) {
+      found = true;
+    }
+    return found;
+  }
+
+  /**
+   * Check the applied properties are set in the pod.
+   *
+   * @param delayTime - time to set delay to extend webapp opened session.
```

Comment:
this does not make sense - what is this parameter for?

Reply:
fixed description